src/SimpleNameRegister.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+///@title An on-chain name registry
+///@author Calnix
+contract SimpleNameRegister {
+    
+    /// @notice map a name to an address to identify current holder 
+    mapping (string => address) public holder;    
+
+    /// @notice emit an event when a name is registered or released
+    event Register(address indexed holder, string name);
+    event Release(address indexed holder, string name);
+
+    /// @notice user can register an available name
```

Comment:
an important part of natspec is describing each `@param`  Try to be creative in the description.  Just repeating the variable name is not great.  Think of it if you were reading this, what extra info might help you?  It's not easy, tbh I would struggle to find a better description than ```@param name The string that will be registered.``` or something...

Reply:
Understood, thanks for the suggestion.